src/main/java/ccm/CcmDemsAdapter.java

```diff
@@ -601,21 +601,22 @@ public void process(Exchange exchange) {
           .log(LoggingLevel.DEBUG, "${exception.message}")
           .setHeader(Exchange.HTTP_RESPONSE_CODE, simple("${exception.statusCode}"))
           .transform(exceptionMessage())
+          .stop()
         .endChoice()
-        .otherwise()
-        .log(LoggingLevel.DEBUG,"DEMS case updated.")
-        .setProperty("courtCaseId", jsonpath("$.id"))
-        .setBody(simple("${exchangeProperty.CourtCaseMetadata}"))
-        .split()
-          .jsonpathWriteAsString("$.accused_persons")
-          .setHeader("key", jsonpath("$.identifier"))
-          .setHeader("courtCaseId").simple("${exchangeProperty.dems_case_id}")
-          .log(LoggingLevel.DEBUG,"Updating accused participant ...")
-          .log(LoggingLevel.DEBUG,"Participant key = ${header.key}")
-          .to("direct:processAccusedPerson")
-          .log(LoggingLevel.DEBUG,"Accused participant updated.")
-        .end()
-        .endChoice()
+      .end()
```

Comment:
Hi Sweta, could you tackle this differently?  I believe we can resolve the defect if we simply move the entire original code into the doTry() block.